ebu_tt_live/examples/config/user_input_producer_handover.json

```diff
@@ -0,0 +1,27 @@
+{
+	"nodes" :
```

Comment:
becuse condifman does not have list support. But I am sure I have explained this one.